static/app/views/organizationLayout.tsx

```diff
@@ -30,30 +24,20 @@ const OrganizationHeader = HookOrDefault({
 /**
  * Renders the organization page layout
  */
-function OrganizationLayout({includeSidebar, children}: OrganizationLayoutProps) {
+function OrganizationLayout({children}: OrganizationLayoutProps) {
   const {organization, loading, error, errorType} = useLegacyStore(OrganizationStore);
   useEnsureOrganization();
 
   if (loading) {
     return <LoadingTriangle>{t('Loading data for your organization.')}</LoadingTriangle>;
   }
 
-  const mainBody = (
-    <SentryDocumentTitle noSuffix title={organization?.name ?? 'Sentry'}>
-      <div className="app">
-        {organization && <OrganizationHeader organization={organization} />}
-        {includeSidebar && <Sidebar organization={organization ?? undefined} />}
-        {children}
-      </div>
-    </SentryDocumentTitle>
-  );
-
   if (error) {
     const errorBody =
       errorType === ORGANIZATION_FETCH_ERROR_TYPES.ORG_NO_ACCESS ? (
-        // We can still render when an org can't be loaded due to 401. The
-        // backend will handle redirects when this is a problem.
-        mainBody
+        <Alert type="error" data-test-id="org-access-error">
+          {t('You do not have access to this organization.')}
+        </Alert>
```

Comment:
There's no reason to render the body without the organization loaded. afaict, we also don't usually return 401 for an Organization?